ProjFS.Mac/PrjFSKext/public/PrjFSCommon.h

```diff
@@ -34,6 +34,22 @@ enum PrjFSServiceUserClientType
     UserClientType_Log,
 };
 
+template <typename T>
+T Clamp(T value, T min, T max)
```

Comment:
What's this used for? I don't see any hits when searching this page.

Reply:
Thanks for pointing that out.  I stopped using `Clamp` in the latest set of changes and so it can be removed.